### PR TITLE
Sync tries progress

### PR DESCRIPTION
**Pull request by iulianpascalau**
- added sync tries progress information prints

### CODE REVIEW

File: data/interface.go [hunk 1]
```diff
@@ -264,3 +264,13 @@ type MiniBlockInfo struct {
 	SenderShardID uint32
 	Round         uint64
 }
+
+// SyncStatisticsHandler defines the method for a component able to store the sync statistics for a trie
```
Comment (SebastianMarian) [inline data/interface.go:268]: methods

File: data/syncer/baseAccountsSyncer.go [hunk 2]
```diff
@@ -102,6 +104,18 @@ func (b *baseAccountsSyncer) GetSyncedTries() map[string]data.Trie {
 	return clonedMap
 }
 
+func (b *baseAccountsSyncer) printStatistics(tss data.SyncStatisticsHandler, ctx context.Context) {
```
Comment (SebastianMarian) [inline data/syncer/baseAccountsSyncer.go:107]: ssh instead tss

File: data/interface.go (hunk 1 above)
Comment (iulianpascalau) [reply data/interface.go:268]: done

File: data/syncer/baseAccountsSyncer.go (hunk 2 above)
Comment (iulianpascalau) [reply data/syncer/baseAccountsSyncer.go:107]: done

**Review by mihaib79 — DISMISSED** [review]

**Review by mihaib79 — APPROVED** [review]

**Review by SebastianMarian — APPROVED** [review]

**Pull request merged**